debug/site.js

```diff
@@ -9,7 +9,8 @@ var map = new mapboxgl.Map({
     hash: true
 });
 
-map.addControl(new mapboxgl.Navigation());
+// map.addControl(new mapboxgl.Navigation());
+map.addControl(new mapboxgl.Geolocate());
```

Comment:
Per #1929 / #1554 I am going to make the button layout more flexible.  This will involve splitting up the Navigation control and moving some of what it does into a container class to manage the screen regions.  Developers should be able to pick and choose which navigation controls they want to appear on the map, and this geolocation button is no different.